### PR TITLE
9827 686 confirmation page

### PR DESCRIPTION
**Pull request by rianfowler**
This is ready for review @jbalboni @peggygannon 
download-1

### CODE REVIEW

File: content/pages/disability-benefits/686/dependent-status.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ entryname: 686-dependent-status
 layout: page-react.html
 ---
 <div id="main">
+  <nav class="va-nav-breadcrumbs">
```
Comment (rianfowler) [inline content/pages/disability-benefits/686/dependent-status.md]: Not sure if this is the way to add the breadcrumbs.

File: src/js/disability-benefits/686/containers/ConfirmationPage.jsx [hunk 2]
```diff
@@ -21,29 +21,58 @@ export class ConfirmationPage extends React.Component {
   }
 
   render() {
+    /*
     const { submission, data } = this.props.form;
     const { response } = submission;
     const name = data.veteranFullName;
```
Comment (rianfowler) [inline src/js/disability-benefits/686/containers/ConfirmationPage.jsx]: Added a [comment](https://app.zenhub.com/workspace/o/department-of-veterans-affairs/vets.gov-team/issues/9827#issuecomment-384287289)

**Comment by jbalboni** [comment]
@rianfowler Let me know if you want me to review this now, or if there are content changes you need to incorporate from that other ticket.

**Review by peggygannon — COMMENTED** [review]
@rianfowler Hi. ok, I made my copy edits directly into the file. Could you please take a look and make sure I didn't mess anything up? 
Also, can you please post a screenshot with the updates, so i can check that my changes look ok? Thank you!

**Comment by rianfowler** [comment]
@peggygannon 
download-2

**Review by peggygannon — APPROVED** [review]
@rianfowler thanks. Looks good.

**Review by jbalboni — APPROVED** [review]

**Pull request merged**